axelrod/strategies/titfortat.py

```diff
@@ -82,3 +82,59 @@ class AntiTitForTat(Player):
 
     def strategy(self, opponent):
         return 'D' if opponent.history[-1:] == ['C'] else 'C'
+
+
+class Shubik(Player):
+    """
+    Plays like Tit-For-Tat with the following modification. After 
+    each retaliation, the number of rounds that Shubik retaliates
+    increased by 1.
+    """
+
+    name = 'Shubik'
+    memoryone = False # Responses depend on more than just the last round
+
+    def __init__(self):
+        Player.__init__(self)
+        self.is_retaliating = False
+        self.retaliation_length = 0
+        self.retaliation_remaining = 0
+
+    def _refresh_retaliation(self, opponent):
+        """Reset the retaliation counter based on the history"""
+        self.retaliation_length = 0
```

Comment:
Could this be made a bit more succinct, something like:

```
self.retaliation_remaining = zip(self.history, opponent.history).count(('C', 'D'))
``` 

Not terribly important but the above is probably faster...